### PR TITLE
fix: allow to delete tests

### PR DESCRIPTION
**Pull request by exu**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: cmd/kubectl-testkube/commands/tests/deleteall.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	"github.com/kubeshop/testkube/pkg/ui"
+	"github.com/spf13/cobra"
+)
+
+func NewDeleteTestsCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "delete <testName>",
+		Short: "Delete tests",
+		Long:  `Delete tests by name`,
+		Args:  validator.TestName,
```
Comment (vsukhin) [inline cmd/kubectl-testkube/commands/tests/deleteall.go:14]: Should be changed for all tests, we don't need name then

File: pkg/api/v1/client/direct.go [hunk 2]
```diff
@@ -553,6 +553,25 @@ func (c DirectScriptsAPI) DeleteTest(name, namespace string) (err error) {
...
+		return fmt.Errorf("prepare request error: %w", err)
+	}
+	resp, err := c.client.Do(req)
+	if err != nil {
+		return fmt.Errorf("client.Do error: %w", err)
+	}
+	defer resp.Body.Close()
+
+	if err := c.responseError(resp); err != nil {
+		return fmt.Errorf("api/delete-test returned error: %w", err)
```
Comment (vsukhin) [inline pkg/api/v1/client/direct.go:569]: delete-tests?

**Review by vsukhin — COMMENTED** [review]
couple minor notes

File: cmd/kubectl-testkube/commands/tests/deleteall.go (hunk 1 above)
Comment (exu) [reply cmd/kubectl-testkube/commands/tests/deleteall.go]: good catch! fixed

File: pkg/api/v1/client/direct.go (hunk 2 above)
Comment (exu) [reply pkg/api/v1/client/direct.go]: good catch! fixed

**Review by vsukhin — APPROVED** [review]

**Pull request merged**